compiler/rustc_abi/src/layout.rs

```diff
@@ -97,16 +99,17 @@ impl<Cx: HasDataLayout> LayoutCalculator<Cx> {
     }
 
     pub fn univariant<
-        'a,
+        'fields,
```

Comment:
Huh... is it possible for us to just require `F: Copy` rather than passing around `&'fields F` everywhere? It seems a bit redundant.

Reply:
Yes that's possible and much cleaner. Idk why I tried so hard to make lifetimes happen here :upside_down_face: